src/tests/ftest/util/agent_utils.py

```diff
@@ -114,28 +115,24 @@ def run_agent(basepath, server_list, client_list=None):
     daos_agent_bin = os.path.join(build_vars["PREFIX"], "bin", "daos_agent")
 
     for client in client_list:
-        cmd = [
-            "ssh",
-            client,
-            daos_agent_bin,
-            "-i"
-        ]
-
-        p = subprocess.Popen(cmd,
-                             stdout=subprocess.PIPE,
-                             stderr=subprocess.PIPE)
-        sessions[client] = p
+        cmd = ["ssh", client, "{} -i".format(daos_agent_bin)]
```

Comment:
Completely optional - in #793 I was able to remove the too-many-locals disabled linting error with this:

```
sessions[client] = subprocess.Popen(
    ["ssh", client, "{} -i".format(daos_agent_bin)],
    stdout=subprocess.PIPE,
    stderr=subprocess.PIPE)
```

Reply:
Actually, that _too-many-locals disabled linting_ was prior to my editing of this file and I must have already eliminated enough locals as I was able to remove those without the suggested change.  But I don't dislike the suggestion so implemented it anyway.  :-)